recipes/osqp/all/conanfile.py

```diff
@@ -18,12 +18,29 @@ class OsqpConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "printing": [True, False],
+        "profiling": [True, False],
+        "ctrlc": [True, False],
+        "dfloat": [True, False],
+        "dlong": [True, False],
+        "coverage": [True, False],
+        "mkl_paradisio": [True, False]
     }
     default_options = {
         "shared": False,
         "fPIC": True,
+        "printing": True,
+        "profiling": True,
+        "ctrlc": True,
```

Comment:
Has been renamed to `interrupt` in upcoming 1.0 version. Might be worth renaming here as well. https://github.com/osqp/osqp/blob/master/CMakeLists.txt#L83